src/lib/routes/admin-api/feature.js

```diff
@@ -49,11 +49,15 @@ class FeatureController extends Controller {
     }
 
     async getAllToggles(req, res) {
-        const features = await this.featureService.getFeatures(
-            req.query,
-            fields,
-        );
-        res.json({ version, features });
+        try {
```

Comment:
would it make sense to move all these "try/catch" calls to the "base-controller" class?

Reply:
That would then probably mean adding a error-handling-middleware, cause the controller base class methods, doesn't actually have access to req or res

Reply:
hmz. I was not aware of that.

Reply:
Let leave it like this for now then. Could be that the controller should have access to req/res, on the other hand, it then starts to smell like a custom middleware.

Reply:
Well, seeing as how we're calling the same util/handleErrors method for almost all of them, writing a custom error-handling middleware might be just what we want.